package.json

```diff
@@ -57,20 +57,22 @@
     "eslint": "^6.1.0",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jest": "^22.16.0",
```

Comment:
I'm not using this directly... not sure how to configure it. cc @ljharb @ahuth WDYT?

Reply:
fine if you want to use it; there's no public airbnb for doing so tho

Reply:
I'm cool with it if we like the rules this gives us. I'm not familiar with what they are, though